cmd/sos_upload.go

```diff
@@ -27,9 +27,8 @@ type fileToUpload struct {
 	contentType string
 }
 
-// uploadCmd represents the upload command
 var sosUploadCmd = &cobra.Command{
-	Use:     "upload <bucket name> <local file path>+",
+	Use:     "upload BUCKET FILE...",
```

Comment:
Shouldn't the `Short` help mention the file instead of the object for consistency:
```suggestion
	Use:     "upload BUCKET FILE...",
	Short: "Upload a file into a bucket",
```